sopel/plugins/rules.py

```diff
@@ -783,6 +804,13 @@ def kwargs_from_callable(cls, handler):
             'rate_limit': getattr(handler, 'rate', 0),
             'channel_rate_limit': getattr(handler, 'channel_rate', 0),
             'global_rate_limit': getattr(handler, 'global_rate', 0),
+            'user_rate_message': getattr(handler, 'user_rate_message', None),
```

Comment:
You know you want to, for consistency. 😏

```suggestion
            'user_rate_message': getattr(
                handler, 'user_rate_message', None),
```